torch/csrc/jit/codegen/cuda/glfdc/expr.cpp

```diff
@@ -0,0 +1,65 @@
+#include <torch/csrc/jit/codegen/cuda/glfdc/expr.h>
+
+#include <c10/util/Exception.h>
+
+using namespace torch::jit::fuser::cuda::glfdc;
```

Comment:
nitpick: we'd want to put the implementations in namespace `torch::jit::fuser::cuda::glfdc` instead of pulling the symbols into `::`